cluster-autoscaler/vendor/k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1beta1/types.go

```diff
@@ -197,7 +197,7 @@ const (
 	// ProvisioningClassCheckCapacity denotes that CA will check if current cluster state can fulfill this request,
 	// and reserve the capacity for a specified time.
 	ProvisioningClassCheckCapacity string = "check-capacity.autoscaling.x-k8s.io"
-	// ProvisioningClassAtomicScaleUp denotes that CA try to provision the capacity
+	// ProvisioningClassBestEffortAtomicScaleUp denotes that CA try to provision the capacity
 	// in an atomic manner.
-	ProvisioningClassAtomicScaleUp string = "atomic-scale-up.autoscaling.x-k8s.io"
```

Comment:
I will be updating vendor when cutting patch release, so this change will get overridden.

Reply:
Sure, I'm adding this change for test to pass

Reply:
Oh, right, this we have a `replace` directive there pointing to local files, so it should be fine.